libs/gltfio/src/ResourceLoader.cpp

```diff
@@ -381,13 +381,19 @@ bool ResourceLoader::loadResources(FFilamentAsset* asset, bool async) {
     // tangent generation.
     decodeDracoMeshes(asset);
 
-    // To be robust against the glTF conformance suite, we optionally ensure that skinning weights
-    // sum to 1.0 at every vertex. Note that if the same weights buffer is shared in multiple
-    // places, this will needlessly repeat the work. In the future we would like to remove this
-    // feature, and instead simply require correct models. See also:
-    // https://github.com/KhronosGroup/glTF-Sample-Models/issues/215
-    if (pImpl->mNormalizeSkinningWeights) {
-        normalizeSkinningWeights(asset);
+    // Normalize skinning weights, then "import" each skin into the asset by building a mapping of
+    // skins to their affected entities.
+    if (gltf->skins_count > 0) {
+        if (pImpl->mNormalizeSkinningWeights) {
+            normalizeSkinningWeights(asset);
```

Comment:
BTW if this causes performance issues we should be able to use the job system and write the code in the way that will produce vectorized code.

Reply:
Yes this would be easy to do since we already use jobs for bounding boxes and tangent gen.  Something for the future.